content/docs/guides/converting-gatsby-to-tina.mdx

```diff
@@ -116,155 +121,83 @@ export default defineConfig({
   },
   media: {
     tina: {
--     mediaRoot: "",
-+     mediaRoot: "images",
+-      mediaRoot: "",
++      mediaRoot: "images"
 -     publicFolder: "public",
 +     publicFolder: "static",
 
     },
 // ...
 ```
 
-## Setting up TinaMarkdown
-
-With the project set up to use TinaCMS, we still need to switch from Gatsby’s MDX parser to TinaCMS’s. Begin by modifying the GraphQL query in gatsby-node.js to return raw markdown:
+Next we'll add the existing frontmatter fields to our existing frontmatter fields to our schema.
 
 ```diff
-  const result = await graphql(`
-    {
-      allMdx(sort: { frontmatter: { date: ASC } }, limit: 1000) {
-        nodes {
-          Id
-+         body
-// ...
-```
-
-Using the raw markdown from `body` we can generate an [Abstract Syntax Tree (AST)](https://tina.io/docs/editing/markdown/#rendering-content-with-tinamarkdown) that we can use with TinaMarkdown.
-
-> Tina converts raw markdown into Abstract Syntax Trees containing the HTML needed to structure the page at build time.
+schema: {
+    collections: [
+      {
+        ui: {
+          router: ({ document }) => {
+            return document._sys.breadcrumbs[0]
+          },
+        },
+        name: "post",
+        label: "Posts",
+        format: "mdx",
+        path: "content/blog",
+        fields: [
+          {
+            type: "string",
+            name: "title",
+            label: "Title",
+            isTitle: true,
+            required: true,
+          },
++          {
++            type: "datetime",
++            name: "date",
++            label: "Date",
++          },
+          {
+            type: "rich-text",
+            name: "body",
+            label: "Body",
+            isBody: true,
+          },
++          {
++            type: "string",
++            name: "description",
++            label: "Description",
++          },
 
-\
-To do this, we'll import TinaCMS's markdown parser.
-Add this import to the top of `gatsby-node.js`.
+        ],
 
-```diff
-+ const { parseMDX } = require("@tinacms/dist")
 ```
 
-Inside `onCreateNode`, we'll parse out the markdown and serialize the result as a string that can be returned from our page query.
+### Updating your images
 
-```diff
-exports.onCreateNode = ({ node, actions, getNode }) => {
-  const { createNodeField } = actions
-  if (node.internal.type === `Mdx`) {
-    const value = createFilePath({ node, getNode })
-+   createNodeField({
-+     name: `tinaMarkdown`,
-+     node,
-+     value: JSON.stringify(parseMDX(node.body)),
-+   })
```

Comment:
**question:** isn't this should be inside code block?

Reply:
This part's being removed, I'm not sure what you mean.

Reply:
I misunderstood the markdown here, please disregard